src/shared/views/RecommendedCards.js

```diff
@@ -81,6 +81,7 @@ type Props = {
     firebase: Object,
     navigation: NavigationProp,
     projects: Array<OrderedProject>,
+    teamId: ?string,
```

Comment:
what does the `?` stand for?

Reply:
It's `flow`'s syntax. Basically means it's optional, ie, can be string or undefined.